### DISCUSSION TITLE
Why is my commits grayed and not available in contribution

### DISCUSSION THREAD

**Discussion by IsraelAbebe** [community/community#22584] [New to GitHub]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/70b71c5d84f80f477ccbf6cc6e5ba23629602664.png" data-download-href="/uploads/short-url/g57MXTjsORhVjjeHeIS8yKTnGKw.png?dl=1" title="Screenshot from 2018-11-28 15-13-35.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/15804692/181072079-417c6508-991a-42e5-81d6-d6070fe0711c.png" alt="Screenshot from 2018-11-28 15-13-35.png" data-base62-sha1="g57MXTjsORhVjjeHeIS8yKTnGKw" width="690" height="314" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/70b71c5d84f80f477ccbf6cc6e5ba23629602664_2_690x314.png, https://aws1.discourse-cdn.com/github/original/1X/70b71c5d84f80f477ccbf6cc6e5ba23629602664.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/70b71c5d84f80f477ccbf6cc6e5ba23629602664.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/70b71c5d84f80f477ccbf6cc6e5ba23629602664_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot from 2018-11-28 15-13-35.png</span><span class="informations">790×360 26.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Tried configuring user.name and email but it didnt work for me </p>

**Comment by Wabri** [comment, marked as answer]
<p>Maybe the problem is the space between Israel and Abebe, in the user.name config you need the same github’s username. So you have to change in your local machine the user.name config with: IsraelAbebe.</p>
<p>Hope to solve your problem</p>
<p>-Gabriele-</p>
